CONTRIBUTING.md

```diff
@@ -18,16 +18,7 @@ There are a couple of guidelines we suggest sticking to:
 
 ## Code Style
 
-Current policy is to only update code to the standard style when changing a substantial portion of it, but **please** do this in a separate commit. Blocks of code can be passed through astyle
```

Comment:
Having this duplicated here didn't really add any value, if you want to read about styling you can just read CODE_STYLE.txt